pkgs/development/python-modules/pygame/default.nix

```diff
@@ -75,5 +76,7 @@ buildPythonPackage rec {
     license = licenses.lgpl21Plus;
     maintainers = with maintainers; [ emilytrau ];
     platforms = platforms.unix;
+    # fatal error: longintrepr.h: No such file or directory.
+    broken = pythonAtLeast "3.11"; # At 2022-02-27
```

Comment:
How about `disabled = pythonAtLeast "3.11"`?

Reply:
`nixpkgs-review` is not showing it as marked broken. Idk, are you sure this is better?

Reply:
I would rather have nixpkgs-review show it as `marked broken` instead of silent failure.

Reply:
I have reversed change to `disabled`. (So I don't know if you mark if conversation is resolved is up to date.)